src/ert/config/observations.py

```diff
@@ -455,15 +447,14 @@ def _handle_general_observation(
                 obs_key,
             ) from err
         if not isinstance(config_node, GenDataConfig):
-            warnings.warn(
+            ConfigWarning.ert_formatted_warn(
                 ConfigWarning.with_context(
```

Comment:
Maybe pull call to `ConfigWarning.with_context` inside of `ert_formatted_warn` ?

Reply:
That is already on its way. 👍

Reply:
:+1: